autoload/db.vim

```diff
@@ -11,6 +11,15 @@ function! s:expand(expr) abort
   return exists('*DotenvExpand') ? DotenvExpand(a:expr) : expand(a:expr)
 endfunction
 
+" Hide pointless `No matching autocommands` in Vim
+if !has('nvim')
+  augroup db_dummy_autocmd
+    autocmd!
+    autocmd User DBQueryStart "
+    autocmd User DBQueryFinished "
```

Comment:
Maybe name these `DBQueryPre` and `DBQueryPost` to better match Vim's naming conventions?

Reply:
Yeah that works. Fixed.